plot-doc/plot/scribblings/renderer2d.scrbl

```diff
@@ -557,6 +557,38 @@ Corresponds with @(racket polar).
 @history[#:changed "7.9" "Added support for pictures for #:label"]
 }
 
+@defproc[(violin
+          [x real?]
+          [ys (sequence/c real?)]
+          [y-min (or/c rational? #f) #f] [y-max (or/c rational? #f) #f]
+          [#:x-min x-min (or/c rational? #f) #f] [#:x-max x-max (or/c rational? #f) #f]
+          [#:samples samples (and/c exact-integer? (>=/c 2)) (line-samples)]
+          [#:color color plot-color/c (interval-color)]
+          [#:style style plot-brush-style/c (interval-style)]
+          [#:line1-color line1-color plot-color/c (interval-line1-color)]
+          [#:line1-width line1-width (>=/c 0) (interval-line1-width)]
+          [#:line1-style line1-style plot-pen-style/c (interval-line1-style)]
+          [#:line2-color line2-color plot-color/c (interval-line2-color)]
+          [#:line2-width line2-width (>=/c 0) (interval-line2-width)]
+          [#:line2-style line2-style plot-pen-style/c (interval-line2-style)]
+          [#:alpha alpha (real-in 0 1) (interval-alpha)]
+          [#:label label (or/c string? pict? #f) #f]
+          [#:invert? invert? boolean? #f]
+          [#:bandwidth bandwidth (or/c real? #f) #f]
+          ) renderer2d?]{
+Draws a violin plot centered at @racket[x].
```

Comment:
When updating this documentation, can you please make the "violin" a link to https://en.wikipedia.org/wiki/Violin_plot